### ISSUE TITLE
Make GRNsight website compatible with screen readers for ADA accessibility

### ISSUE THREAD

**Issue by kdahlquist** [dondi/GRNsight#793]
There is a lot of discussion of making websites ADA accessible on NPR these days.  Although our program is a visualization program and would not necessarily be useful for the visually impaired, we can still make our website compatible with screen readers.

@dondi, this might be a good project for your interaction design course.

**Comment by dondi** [comment]
Yes, universal accessibility is indeed transitioning from a best practice to a must-practice. We can queue this up for someone in the near future.

**Comment by kdahlquist** [comment]
Will become easier upon transition to React.
